### DISCUSSION TITLE
Erro Not Found (Apache/2.4.46 (Win64) OpenSSL/1.1.1h PHP/7.4.12 Servidor em x.x.x.x Porta 80)

### DISCUSSION THREAD

**Discussion by Brunodias02** [RamonSilva20/mapos#2552] [Q&A]
Boa noite,

Não estou conseguindo abrir o MAPOS no navegador.

![image](https://github.com/user-attachments/assets/b2f7b74d-0713-43c5-beb6-7a902d7d998b)

**Comment by Marc3lo-sa** [comment]
Talvez o problema esteja relacionado com versão do xampp instalada, ou conflito com outra aplicação que utiliza a mesma porta 80
